Cargo.toml

```diff
@@ -17,3 +17,7 @@ num = "0.1.30"
 
 [features]
 nightly = []
+
+# Unstable APIs that have not yet
+# proven their utility.
+unstable = []
```

Comment:
You need to set up your editor/gitconfig to insert new lines at the end of your files :3